### PR TITLE
Integrate examples to homepage

### PR DESCRIPTION
**Pull request by mbssantos**
Part of #19 

[WIP]:
- [x] Wealth of Nations charts circles are too wide
- [x] Stacked chart not aligned with y-axis

### CODE REVIEW

File: site/src/examples/stacked/stacked.js [hunk 1]
```diff
@@ -1,85 +1,83 @@
...
+    })
+    .mainValue(function (d) {
+      return d[1];
+    })
+    .baseValue(function (d) {
+      return d[0];
+    })
+    .decorate(function (selection, data, index) {
+      selection.enter().attr("fill", color(series[count++].key));
+      count = count >= 5 ? 0 : count;
```
Comment (mbssantos) [inline site/src/examples/stacked/stacked.js]: Horrible hack because index is always 0 although it runs 5 times

Comment (chrisprice) [inline site/src/examples/stacked/stacked.js]: I think I have a solution, give me a shout when you get a minute

**Comment by mbssantos** [comment]
@ColinEberhardt @chrisprice All sorted.
A few blemishes to iron out such as color and scale on axes

**Comment by ColinEberhardt** [comment]
Awesome! I'll grab my iron :iron-emoji:

**Comment by ColinEberhardt** [comment]
Thanks @alisd23 @mbssantos - I've done a few minor tweaks, squashed, and will merged via #77

**Pull request closed without merging**